test_runner/src/main/kotlin/ftl/cli/firebase/CancelCommand.kt

```diff
@@ -0,0 +1,29 @@
+package ftl.cli.firebase
+
+import ftl.run.TestRunner
+import kotlinx.coroutines.experimental.runBlocking
+import picocli.CommandLine
+
+@CommandLine.Command(
+        name = "cancel",
+        sortOptions = false,
+        headerHeading = "",
+        synopsisHeading = "%n",
+        descriptionHeading = "%n@|bold,underline Description:|@%n%n",
+        parameterListHeading = "%n@|bold,underline Parameters:|@%n",
+        optionListHeading = "%n@|bold,underline Options:|@%n",
+        header = ["Cancels the last Firebase Test Lab run"],
+        description = ["""Selects the most recent run in the results/ folder.
+Reads in the matrix_ids.json file. Cancels any incomplete matrices.
+"""]
+)
+class CancelCommand : Runnable {
+    override fun run() {
```

Comment:
I consolidated the cancel to just one command because calling it will just cancel the matrices in last run folder. If we want to specify whether we want to cancel an iOS or Android last run, we will need to structure our results folder differently, like have `results-ios` and `results-android` folders or prefix our run folders with `ios-` or `android-`.